src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -265,6 +292,11 @@ type MessageDbContext(connection : MessageDbConnection, batchOptions : BatchOpti
     member _.LoadBatched streamName log tryDecode : Async<StreamToken * 'event array> = async {
         let! version, events = Read.loadForwardsFrom log connection.ReadRetryPolicy connection.ReadConnection batchOptions.BatchSize batchOptions.MaxBatches streamName 0L
         return Token.create version, Array.chooseV tryDecode events }
+    member _.LoadLast streamName log tryDecode : Async<StreamToken * 'event array> = async {
```

Comment:
tupled args for members please (feel free to make a PR for the other stores that are also wrong!)
I used to do this back in the day but its against eh F# style guide for good reasons